torch/ao/quantization/fx/utils.py

```diff
@@ -180,7 +180,7 @@ def get_quantize_node_info(activation_post_process: Callable) -> Optional[Tuple[
         quantize_op = torch.quantize_per_tensor_dynamic
         # TODO: get reduce range from observer
         # reduce_range = activation_post_process.reduce_range
-        reduce_range = torch.backends.quantized.engine == "fbgemm"
+        reduce_range = torch.backends.quantized.engine in ("fbgemm", "x86")
```

Comment:
@Xia-Weiwen I noticed that you set a reduced custom range instead of setting `reduce_range=True` when initializing the activation observers in the x86 default qconf. The default `reduce_range` is `False`. But here you set the `reduce_range` to `True` for x86. Would that effectively reduce the range twice? cc @jerryzh168